src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js

```diff
@@ -117,56 +154,61 @@ Summary
   const parseVaccines = records => {
     return `
 ${txtLine}
-Vaccines
+3) Vaccines
 
 This list includes vaccines you got at VA health facilities and from providers or pharmacies in our community care network. It may not include vaccines you got outside our network.
 For complete records of your allergies and reactions to vaccines, review your allergy records in this report.
 
-${records.map(
-      record =>
-        `
+${records
+      .map(
+        record =>
+          `
 ${record.name}
 ${txtLineDotted}
 ${`Date received: ${record.date}`}
 ${`Location: ${record.location}`}
 Provider notes
 ${record.notes.map(note => `${note}`)}
             `,
-    )}
+      )
+      .join('')}
     `;
   };
 
   //  allergies parse
   const parseAllergies = records => {
     return `
 ${txtLine}
-Allergies
+4) Allergies
 
 If you have allergies that are missing from this list, send a secure message to your care team.
-${records.map(
-      record => `
+${records
+      .map(
+        record => `
 ${record.name}
 ${txtLineDotted}
 Date entered: ${record.date}
-Signs and symptoms: ${record.reaction.map(reaction => `${reaction}`)}
+Signs and symptoms: ${record.reaction.map(reaction => `${reaction}`).join('')}
```

Comment:
Don't use `.join('')` on this one as it is a real list. Instead use `.join(', ')`
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/ce064369-b019-47ce-a780-80620941bdae)